game/libda/portaudio.hpp

```diff
@@ -223,6 +234,9 @@ namespace portaudio {
 		  PaStreamCallback* callback = NULL,
 		  void* userData = NULL)
 		{
+			if (output != nullptr) {
```

Comment:
little inconsistency here, you're using nullptr, but elsewhere NULL is used. might be a good idea to put these in the code standard.

Reply:
From what I understand, nullptr is generally to be preferred when working with pointers. So IMO we should update those other NULLs.

Reply:
P.S. I already did it but will open a new PR.